src/Compilers/Core/Desktop/CommandLine/RuleSet/RuleSet.cs

```diff
@@ -209,7 +209,7 @@ private void GetEffectiveIncludesCore(ImmutableArray<string>.Builder arrayBuilde
                 }
 
                 // If this file has already been included don't recurse into it.
-                if (!arrayBuilder.Contains(ruleSet.FilePath, StringComparer.InvariantCultureIgnoreCase))
+                if (!arrayBuilder.Contains(ruleSet.FilePath, StringComparer.OrdinalIgnoreCase))
```

Comment:
This does not exist in CoreFx and seemed incorrect anyways.  This is a file path comparison.